src/nvim/memline.c

```diff
@@ -1277,7 +1277,7 @@ recover_names (
   int num_files;
   int file_count = 0;
   char_u      **files;
-  int i;
+  int i, j;
```

Comment:
Would you mind moving this to local block scope for each `for(int i = 0; ...)` block in this function?